apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -268,18 +274,33 @@ class ReviewTab extends Component {
   render() {
     const {comments, forceRecreateEditorKey, isReadyForReview} = this.state;
 
-    return (
-      <div style={styles.reviewsContainer}>
-        <div style={styles.reviewHeader}>
-          {this.renderReadyForReviewCheckbox()}
+    // channelId is not available on projects where the student has not edited the starter code.
+    // comments cannot be made on projects in this tate.
+    const projectOwnerHasEditedCode = !!getStore().getState().pageConstants
+      .channelId;
+
+    if (projectOwnerHasEditedCode) {
```

Comment:
nit: same here, invert and early return the disabled message if `!projectOwnerHasEditedCode`?